src/main/java/racingcar/domain/Winners.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.domain;
+
+import racingcar.domain.car.Cars;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winners {
+    public static final String AND = ", ";
+    private final List<String> winners;
+
+    public Winners(Cars cars) {
+        int maxLocation = findMaxLocation(cars);
+        List<String> winners = findWinners(cars, maxLocation);
```

Comment:
```suggestion
        this.winners = findWinners(cars, maxLocation);
```

변수를 사용하지 않고 바로 초기화하는 것과 어떠한 차이가 있을까요?

Reply:
winners List를 하나 더 만들면 만들지 않아도 될 불필요한 자원이 쓰일 것 같습니다.